jena-text/pom.xml

```diff
@@ -81,6 +81,35 @@
       <artifactId>lucene-queryparser</artifactId>
     </dependency>
 
+      <dependency>
+          <groupId>org.elasticsearch</groupId>
+          <artifactId>elasticsearch</artifactId>
+      </dependency>
+
+      <dependency>
+          <groupId>org.elasticsearch.client</groupId>
+          <artifactId>transport</artifactId>
+      </dependency>
+
+      <!-- This is required to by pass ES JAR Hell in test environment-->
```

Comment:
The comment seems outdated, wasn't this related to the embedded ES tests that have now been removed?